dynamic_rest/fields/fields.py

```diff
@@ -135,7 +135,7 @@ def bind(self, *args, **kwargs):
                 )
         ):
             self.required = False
-        if 'allow_null' not in self.kwargs and getattr(
+        if 'allow_null' in self.kwargs and getattr(
```

Comment:
Ah, nm on that - `not in` seems correct to dependents. Just wondering about the failing unit test then.